<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTypeDialog</class>
 <widget class="QDialog" name="EditTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar TIM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_21">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Nome do tipo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="QLabel" name="prefixLabel">
         <property name="text">
          <string>Prefixo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="prefixEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Recursivo?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="parentChildLabel">
         <property name="text">
          <string>Pai -&gt; Filho</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="minCardChildLabel">
           <property name="text">
            <string>Nº mínimo de filhos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minCardChildEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="maxCardChildLabel">
           <property name="text">
            <string>Nº máximo de filhos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCardChildEdit">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="childDependencyCheckBox">
         <property name="text">
          <string>Filho é dependente?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="childParentLabel">
         <property name="text">
          <string>Filho -&gt; Pai</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="minCardParentLabel">
           <property name="text">
            <string>Nº mínimo de pais</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minCardParentEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <item>
          <widget class="QLabel" name="maxCardParentLabel">
           <property name="text">
            <string>Nº máximo de pais</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCardParentEdit">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="parentDependencyCheckBox">
         <property name="text">
          <string>Pai é dependente?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
